src/sentry/similarity/__init__.py

```diff
@@ -57,21 +60,31 @@ def get_frame_attributes(frame):
     return attributes
 
 
-features = FeatureSet(
-    MinHashIndex(
-        redis.clusters.get(
-            getattr(
+def _make_index(cluster=None):
+    if not cluster:
+        try:
+            cluster_id = getattr(
```

Comment:
I don't think this needs to be in the `try` block here.

Reply:
f0d0b3cd161b711cc5dd25fda0c2c0feee28b80f